simphony/cuds/lattice.py

```diff
@@ -25,30 +24,33 @@ def __init__(self, index, data=None):
             self.data = DataContainer(data)
 
 
-class Lattice(ABCLattice):
-    """
-    A Bravais lattice;
-    stores references to data containers (node related data).
+class MetaLattice(ABCLattice):
+    """ A Bravais lattice metadata structure.
 
-    Parameters
-    ----------
-    name : str
-    type : str
+    Can be instantiated. Initializes lattice attributes, but not
+    DataContainers.
+
+    Attributes:
+    -----------
+    name: string
+        name of the lattice
+    type: string
         Bravais lattice type (should agree with the base_vect below).
-    base_vect : D x float
+    base_vect: D x float
```

Comment:
update link to the numpy docstring guideline https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt